ide/idea/workspace.xml

```diff
@@ -227,6 +227,7 @@
                                           -Dcassandra.logdir=$PROJECT_DIR$/data/logs
                                           -Dcassandra.reads.thresholds.coordinator.defensive_checks_enabled=true
                                           -Dcassandra.storagedir=$PROJECT_DIR$/data
+                                          -Dcassandra.triggers_dir=$PROJECT_DIR$/conf/triggers
```

Comment:
This needs to be here otherwise running Cassandra from IDEA will not load that trigger which was installed there by `ant install` from `examples/triggers`.